WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController.swift

```diff
@@ -1801,7 +1784,7 @@ private extension NotificationsViewController {
     }
 
     func resetLastSeenTime() {
-        lastSeenTime = nil
+        viewModel.lastSeenTime = nil
     }
```

Comment:
That is indeed the case and it is being used from outside. I thought to keep the changes minimally intrusive going forward as well to avoid regression. But I suppose for this specific case at least, it wouldn't cause anything to just do what you've suggested